lib/modules/symbolize_keys.rb

```diff
@@ -0,0 +1,8 @@
+def symbolize_keys(hash)
```

Comment:
There should be a module defined here, not just a floating method. Something like `module MyHelper`. So it should be `lib/modules/my_helper`, where all of the methods are defined.